app/javascript/packs/pw-strength.js

```diff
@@ -96,6 +96,17 @@ function disableSubmit(submitEl, length = 0, score = 0) {
   }
 }
 
+/**
+ * @param {HTMLElement?} element
+ *
+ * @return {string[]|undefined}
+ */
+export function getForbiddenPasswords(element) {
+  try {
+    return JSON.parse(element.dataset.forbiddenPasswords);
```

Comment:
oooo I have not seen `.dataset` I'm just used to `.getAttribute('data-forbidden-passwords')`